lib/chef/azure/commands/enable.rb

```diff
@@ -92,7 +114,24 @@ def install_chef_service
   end
 
   def enable_chef_service
-    @exit_code, error_message = ChefService.new.enable(@chef_extension_root, bootstrap_directory, @azure_plugin_log_location)
+    chef_service_interval = load_chef_service_interval
+
+    if chef_service_interval.empty?
+      @exit_code, error_message = ChefService.new.enable(@chef_extension_root, bootstrap_directory, @azure_plugin_log_location)

```

Comment:
@mwrock We have treated empty input as the action to install `chef service` with the default interval of `30 minutes`. This would help to keep the `extension` behavior (to install `chef service` with 30 minutes interval) consistent and users would not have to change their custom scripts to include `chef_service_interval` option with `30 minutes` interval to install the `chef service` as it was previously doing. For users who don't want `chef service` to be installed, they can explicitly pass `0` as the input for `chef_service_interval` option. Values other than `0` for `chef_service_interval` option will be treated as the action to install `chef service` with the given interval value.